app/config.py

```diff
@@ -52,9 +52,7 @@ class Config(object):
     EXPIRE_CACHE_IN_SECONDS = 600
 
     # Performance platform
-    PERFORMANCE_PLATFORM_ENABLED = os.getenv('PERFORMANCE_PLATFORM_ENABLED') == '1'
-    PERFORMANCE_PLATFORM_URL = 'https://www.performance.service.gov.uk/data/govuk-notify/notifications'
-    PERFORMANCE_PLATFORM_TOKEN = os.getenv('PERFORMANCE_PLATFORM_TOKEN')
+    PERFORMANCE_PLATFORM_ENABLED = False
```

Comment:
shouldn't be moving these vars, or they won't be defined outside of prod and it may crash